exec/java-exec/src/main/java/org/apache/drill/exec/planner/cost/DrillRelMdDistinctRowCount.java

```diff
@@ -80,10 +87,10 @@ public Double getDistinctRowCount(Join rel, RelMetadataQuery mq,
 
   @Override
   public Double getDistinctRowCount(RelNode rel, RelMetadataQuery mq, ImmutableBitSet groupKey, RexNode predicate) {
-    if (rel instanceof DrillScanRelBase) {                  // Applies to both Drill Logical and Physical Rels
+    if (rel instanceof TableScan) {                         // Applies to both Drill Logical and Physical Rels
```

Comment:
Comment should indicate that it applies to Calcite logical table scan also since that seems to be the purpose of this change.

Reply:
Done